metron-interface/metron-alerts/src/app/alerts/alerts-list/tree-view/tree-view.component.ts

```diff
@@ -421,10 +423,13 @@ export class TreeViewComponent extends TableViewComponent implements OnInit, OnC
     if (this.canCreateMetaAlert(group.total)) {
       let confirmationMsg = 'Do you wish to create a meta alert with ' +
                             (group.total === 1 ? ' alert' : group.total + ' selected alerts') + '?';
-      this.metronDialogBox.showConfirmationMessage(confirmationMsg).subscribe((response) => {
-        if (response) {
+      let confirmedSubscription = this.dialogService.confirm(confirmationMsg).subscribe(r => {
+        if (r === 'Confirmed') {
```

Comment:
I suggest to use (r === ConfirmationType.Confirmed) instead of the string literal.